.github/workflows/CI.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  push:
+    branches:
+      - kokkos_develop_testing
+  schedule:
+    - cron:  '0 1 * * *'
```

Comment:
Every day one minute after midnight, ok?

Reply:
Works for me!